pkg/apis/datadoghq/v1alpha1/watermarkpodautoscaler_types.go

```diff
@@ -58,12 +58,14 @@ type WatermarkPodAutoscalerSpec struct {
 	UpscaleForbiddenWindowSeconds int32 `json:"upscaleForbiddenWindowSeconds,omitempty"`
 
 	// Percentage of replicas that can be added in an upscale event.
-	// Parameter used to be a float, in order to support the transition seamlessly, we validate that it is ]0;100] in the code.
-	ScaleUpLimitFactor resource.Quantity `json:"scaleUpLimitFactor,omitempty"`
+	// Parameter used to be a float, in order to support the transition seamlessly, we validate that it is [0;100] in the code.
```

Comment:
Finding this comment a bit confusing, is the `used to be a float` a reference to prior implementation. Should we change this comment for clarity?

Reply:
ah yeah I can lol